### DISCUSSION TITLE
Installation goes wrong - Can someone explain what is wrong with my setup?

### DISCUSSION THREAD

**Discussion by lfb0801** [LazyVim/LazyVim#483] [Q&A] [answered]
I did a clean install of neovim, but am encountering lots of issues.
It is unclear for me what I did wrong, or what steps I need to take to make things work, can someone help?

![image](https://user-images.githubusercontent.com/16375630/227241080-8ead479a-1192-4f60-b81e-fcb3bbb99c5a.png)
![image](https://user-images.githubusercontent.com/16375630/227241676-24fc6be4-b217-4b96-959f-68b9385d8da9.png)
![image](https://user-images.githubusercontent.com/16375630/227241820-d860b0c0-f0b8-4422-ba5e-e39fdd0eb339.png)

**Comment by folke** [comment, marked as answer]
Your Neovim installation seems to be severely broken...

**Reply by lfb0801** [reply]
Yes, I agree with that. Do you have suggestions to make sure this doesn't happen again?

**Reply by folke** [reply]
What do you mean? Your Neovim installation is broken and that's not by lazy or LazyVim. My best guess is that your NEovim runtime directory is not the one for the installed binary. I can't fix incorrect Neovim installs. That's on you

>Do you have suggestions to make sure this doesn't happen again?
??
